aea/cli/generate_wealth.py

```diff
@@ -38,21 +38,28 @@
     required=True,
 )
 @click.argument("url", metavar="URL", type=str, required=False, default=None)
+@click.argument(
+    "password", metavar="PASSWORD", type=str, default=None, required=False,
+)
```

Comment:
since it is used a lot, it could be implemented as "custom Click type". In the future, we may also want to restrict the type of passwords, e.g. see [here](https://en.wikipedia.org/wiki/Password_strength#Guidelines_for_strong_passwords).

Reply:
If my strategy above is applied, would be interesting to delegate the Click parameter type class to call getpass, so we get a pretty neat solution.